src/commands/synthetics/run-test.ts

```diff
@@ -1,42 +1,47 @@
 import chalk from 'chalk'
 import {Command} from 'clipanion'
+import deepExtend from 'deep-extend'
 
-import {parseConfigFile, ProxyConfiguration} from '../../helpers/utils'
+import {parseConfigFile} from '../../helpers/utils'
 import {apiConstructor} from './api'
-import {APIHelper, ConfigOverride, ExecutionRule, LocationsMapping, MainReporter, PollResult, Test} from './interfaces'
+import {APIHelper, CommandConfig, ExecutionRule, LocationsMapping, MainReporter, PollResult, Test} from './interfaces'
 import {DefaultReporter} from './reporters/default'
 import {Tunnel} from './tunnel'
 import {getReporter, getSuites, getTestsToTrigger, hasTestSucceeded, runTests, waitForResults} from './utils'
 
+export const DEFAULT_COMMAND_CONFIG: CommandConfig = {
+  apiKey: '',
+  appKey: '',
+  configPath: 'datadog-ci.json',
+  datadogSite: 'datadoghq.com',
+  fileGlobs: ['{,!(node_modules)/**/}*.synthetics.json'],
+  global: {},
+  pollingTimeout: 2 * 60 * 1000,
+  proxy: {protocol: 'http'},
+  publicIds: [],
+  subdomain: 'app',
+  tunnel: false,
+}
+
 export class RunTestCommand extends Command {
   private apiKey?: string
   private appKey?: string
-  private config = {
-    apiKey: process.env.DATADOG_API_KEY,
-    appKey: process.env.DATADOG_APP_KEY,
-    datadogSite: process.env.DATADOG_SITE || 'datadoghq.com',
-    files: '{,!(node_modules)/**/}*.synthetics.json',
-    global: {} as ConfigOverride,
-    pollingTimeout: 2 * 60 * 1000,
-    proxy: {protocol: 'http'} as ProxyConfiguration,
-    subdomain: process.env.DATADOG_SUBDOMAIN || 'app',
-    tunnel: false,
-  }
+  private config: CommandConfig = JSON.parse(JSON.stringify(DEFAULT_COMMAND_CONFIG))
```

Comment:
The deep copy does not seem to be needed

Reply:
I think it's still good to copy it to avoid mutating the default, even if we don't refer to `DEFAULT_COMMAND_CONFIG` anywhere else for now, we might in the future. But we can use `deepCopy()` in `helpers.ts`

Reply:
`DEFAULT_COMMAND_CONFIG` is used in tests and some tests modify it. I needed to deep copy to avoid confit between tests.